boards/arm/cxd56xx/common/src/cxd56_crashdump.c

```diff
@@ -239,4 +239,5 @@ void board_crashdump(uintptr_t currentsp, void *tcb,
 #if defined(CONFIG_CXD56_RESET_ON_CRASH)
   board_reset_on_crash();
 #endif
```

Comment:
should we change to this?
```suggestion
#else
  return;
#endif
```

Reply:
done